components/member-profile/contribution/index.js

```diff
@@ -10,11 +11,70 @@ const renderPRLinks = (prList) =>
 
 const Contribution = ({ contribution, fullName, imageLink, devUser }) => {
   const {
-    task: { title, startedOn, endsOn, status, purpose, featureUrl },
+    task: { featureUrl },
+    prList
+  } = contribution;
+  const url = featureUrl || prList[0]['url'];
+  const gotoUrl = () => url && window.open(url, '_blank');
+  const urlObj = url && new URL(url);
+  const card = () => (
+    <ContributionCard
+      contribution={contribution}
+      fullName={fullName}
+      imageLink={imageLink}
+      devUser={devUser}
+      url={url}
+      urlObj={urlObj}
+    />
+  );
+  const renderFeatureCard = () => {
+    if (urlObj?.host === 'members.realdevsquad.com') {
+      return (
+        <Link href={urlObj.pathname}>
+          <div className={url && classNames.contributionCard}>{card()}</div>
+        </Link>
+      );
+    } else {
```

Comment:
we don't need an else here since we are returning in the `if` block.
so you can directly return the `div`

```javascript

if (condition) {
 return ()
}

return (the code that you are returning in the else block)
```